src/pre_process/m_variables_conversion.f90

```diff
@@ -245,6 +247,15 @@ subroutine s_initialize_variables_conversion_module() ! -------------------
 
     end subroutine s_initialize_variables_conversion_module ! -----------------
 
+    !>  This procedure calculates the pressure based on energy when
```

Comment:
Ah, but I believe it's alright since that file is already deleted in this version? That comment seems to be in the pre_process version of m_var_conversion which is gone now.